dali/operators/reader/loader/video/frames_decoder.h

```diff
@@ -70,6 +71,23 @@ struct AvState {
   }
 };
 
+/**
+ * @brief Helper representing video file kept in memory. Allows reading and seeking.
+ * 
+ */
+struct MemoryVideoFile {
+  explicit MemoryVideoFile(char *data, int64_t size)
```

Comment:
I'm wondering, whether this struct is designed just to keep the video and optionally read from it, or can it change the file? If it's the former, I'd go with adding `const`, but if it's the latter, I believe the ownership should also be added.
```suggestion
  MemoryVideoFile(const uint8_t *data, int64_t size)
```

Reply:
I did the consts. Same as the above with regards to types.